### ISSUE TITLE
Unable to create a backup for months now. It won't finish loading tracks. Gets stuck at around 60 000 tracks.

### ISSUE THREAD

**Issue by uusseerrnnaammee** [secuvera/SpotMyBackup#60]
Unable to create a backup for months now. It won't finish loading tracks. Gets stuck at around 60 000 tracks. Is this not updated/working anymore or what is the problem..

**Comment by uusseerrnnaammee** [comment]
Yeah now it gets stuck all the time to 160 lists 52xxx songs and I cannot create a backup. Before it might have stopped a bit earlier 130, 145, 152, etc.. but now stops always at 160 lists 52xxx songs..  I wonder what could cause this behavior and is there anything to be done..

**Comment by uusseerrnnaammee** [comment]
still the same issue

**Comment by JambDev** [comment]
Is it possible you could send the backup file you are using?
I only have like 400 songs so I can't replicate this.

**Comment by twttr** [comment]
I've had an issue once with ~9000 tracks, spotify repsonded with 503 to one of the API requests, might also be an issue in your case, seems like script is not handling this.
